OWNERS

```diff
@@ -0,0 +1,5 @@
+assignees:
+- brendandburns
```

Comment:
I misspoke. My comment just applies to the top level OWNERS. Also, we can probably just delete docs/OWNERS because that will be implied by OWNERS